CREDITS

```diff
@@ -0,0 +1,38 @@
+Authors
+=======
+Major contributing authors according to `git shortlog -sn` include:
+
+-Daniel Moran
+-Alvaro Vega
+-Fermín Galán Márquez
+-Daniel Calvo
+-xval
+-Jason Fox
+-a620381
+-Jagatjot Singh
+-Germán Toro del Valle
+-Alvaro Vega Garcia
+-Rafael Gonzalez
+-Tomas Aliaga
+-Álvaro Arranz
+-Fermin Galan Marquez
+-Víctor García Guillén
+-anabelengp
+-Marcos Reyes
+-Cesar Jorge
+-develenv
+-frbattid
+-Ana Belén García
+-Cesar Jorge Martinez
+-Francisco Romero Bueno
+-Maurus Frey
+-Víctor García
+-Drasko DRASKOVIC
+-Jose M. Cantera
+-Morton Fox
+-Pawel Jedruch
+
+For a complete list of contributors, see
+https://github.com/telefonicaid/iotagent-json/graphs/contributors
+
+We would like to thank everyone who contributed to QuantumLeap, also to those who helped discuss and mature the core ideas.
```

Comment:
> We would like to thank everyone who contributed to QuantumLeap,  ...

hmmm... copy-paste boilerplate - should be *IoT Agent for JSON* I guess.

Reply:
I think there is no need of working further on this PR. As I comment [here](https://github.com/telefonicaid/iotagent-json/pull/386#issuecomment-464762167) it's better GE owners add the CREDITS file, as they know how to build the list (this link provide could help to provide more context: https://github.com/Fiware/contribution-requirements/pull/39#discussion_r250975489).

Reply:
okay @fgalan